src/View/Helper/FormHelper.php

```diff
@@ -766,7 +766,10 @@ public function error(string $field, $text = null, array $options = []): string
             }
         }
 
-        return $this->formatTemplate('error', ['content' => $error]);
+        return $this->formatTemplate('error', [
+            'content' => $error,
+            'id' => $field . '-aria',
```

Comment:
This value needs to be passed through `FormHelper::_domId()`, same for the `aria-describedby` option value.

And maybe this should have a more generic suffix, so that it doesn't indicate such a strictly tied purpose.

Reply:
I don't understand how I should do it? For ARIA it is necessary that the element has an ID, and it must have some prefix - because the text field itself has an ID. So if they had the same ID, it would not be valid according to W3C.

Reply:
Since IDs aren't something specifically related to ARIA, I'd suggest to not use `-aria` as a suffix, but something more generic that fits what the element represents, like `-error`.